### PR TITLE
Updates for 3.0 release

### PR DESCRIPTION
**Pull request by speth**
These pages all need to be checked/updated for the 3.0.0 release. The work-in-progress version is live at https://testing.cantera.org. Merging this PR should correspond to the release of Cantera 3.0.0.

# pages/
- [x] affiliated.rst
- [x] community.rst
- [x] dave-goodwin.rst
- [x] governance.rst
- [x] index.html
## pages/compiling/
- [x] config-options-dev.rst
- [x] config-options.rst
- [x] configure-build-dev.rst
- [x] configure-build.rst
- [x] dependencies.rst
- [x] installation-reqs.rst
- [x] scons-config-options-dev.rst
- [x] scons-config-options.rst
- [x] source-code.rst
- [x] special-cases.rst
## pages/documentation/
- [x] dev-docs.html
- [x] glossary.rst
- [x] index.html
### pages/documentation/release_notes/
- [x] v3.0.0.md
## pages/examples/
- [x] index.rst
## pages/install/
- [x] compiling-install.rst
- [x] conda-install.rst
- [x] ~fedora-rhel-install.rst~ *deferred*
- [x] ~freebsd-install.rst~ *deferred*
- [x] ~gentoo-install.rst~ *deferred*
- [x] install.rst
- [x] macos-install.rst
- [x] ~opensuse-install.rst~ *deferred*
- [x] pip-install.rst
- [x] ubuntu-install.rst
- [x] windows-install.rst
## pages/science/
- [x] flames.rst
- [x] index.rst
- [x] kinetics.rst
- [x] phase-thermo.rst
- [x] species-thermo.rst
- [x] thermodynamics.rst
- [x] transport.rst
## pages/science/reactors/
- [x] constpresmolereactor.rst
- [x] constpresreactor.rst
- [x] controlreactor.rst
- [x] customreactor.rst *deleted*
- [x] cvodes.rst
- [x] extensiblereactor.rst
- [x] idealgasconstpresmolereactor.rst
- [x] idealgasconstpresreactor.rst
- [x] idealgasmolereactor.rst
- [x] idealgasreactor.rst
- [x] molereactor.rst
- [x] pfr.rst
- [x] reactors.rst
## pages/tutorials/
- [x] MatlabTutorial.rst
- [x] PythonTutorial.rst
- [x] ck2cti-tutorial.rst *deleted*
- [x] ck2yaml-tutorial.rst
- [x] index.rst
- [x] input-files.rst
- [x] legacy2yaml-tutorial.rst
- [x] thermobuild.rst
- [x] cti/* *deleted*
### pages/tutorials/cxx-guide/
- [x] compiling.rst
- [x] demo1a.cpp
- [x] demoequil.cpp
- [x] equil-example.rst
- [x] factories.rst *(renamed to `kinetics-transport.rst`)*
- [x] factory_demo.cpp *(renamed to `kinetics_transport.cpp`)*
- [x] headers.rst
- [x] index.rst
- [x] simple-example.rst
- [x] thermo.rst
- [x] thermobasic.cpp
- [x] thermodemo.cpp
### pages/tutorials/yaml/
- [x] defining-phases.rst
- [x] phases.rst
- [x] reactions.rst
- [x] species.rst
- [x] yaml-format.rst

### CODE REVIEW

File: pages/science/kinetics.rst
Comment (ischoegl) [inline pages/science/kinetics.rst:1]: Presumably, this closes Cantera/enhancements#142

File: pages/install/conda-install.rst
Comment (ischoegl) [inline pages/install/conda-install.rst:114]: `HighFive` should be added?

**Comment by speth** [comment]
I think these updates for the 3.0 release are complete now, with the exception of the release notes.

**Review by ischoegl — APPROVED** [review]
@speth ... thanks for this massive effort.

File: pages/documentation/release_notes/v3.0.0.md
Comment (ischoegl) [inline pages/documentation/release_notes/v3.0.0.md:10]: Needs update.

File: pages/documentation/release_notes/v3.0.0.md
Comment (ischoegl) [inline pages/documentation/release_notes/v3.0.0.md:107]: Missing yeanment

File: pages/documentation/release_notes/v3.0.0.md
Comment (ischoegl) [inline pages/documentation/release_notes/v3.0.0.md:70]: Also 1594

**Review by ischoegl — COMMENTED** [review]
@speth … I’m sure you’re aware, but I’m adding this review just in case.

**Comment by speth** [comment]
Thanks for looking this over, @ischoegl. I was leaving this in "draft" mode until we merge Cantera/cantera#1599 and tag the release, mainly to make sure that we don't merge it before then. All of the content is "ready for review", though the contents of the release notes especially are subject to change until then. Final updates to the date, PR list, and contributor list in the release notes are all already on my radar, but I'm glad someone else is keeping an eye on these things :eyes:.

**Pull request merged**
